modules/account-map/modules/iam-roles/variables.tf

```diff
@@ -4,12 +4,6 @@ variable "privileged" {
   default     = false
 }
 
-variable "global_tenant_name" {
-  type        = string
-  description = "Global tenant name"
-  default     = null
-}
-
```

Comment:
I would prefer to leave this. So far we haven't had a client for whom this was not sufficient.